src/DssSpell.t.sol

```diff
@@ -433,14 +451,14 @@ contract DssSpellTest is DssSpellTestBase {
 
         _vote(address(spell));
         _scheduleWaitAndCast(address(spell));
-        assertTrue(spell.done());
+        assertTrue(spell.done(), "TestError/spell-not-done");
         for (uint256 i = 0; i < yanks.length; i++) {
             // Test stream.fin is set to the current block after the spell
             assertEq(vest.fin(yanks[i].streamId), block.timestamp, "testYankDAI/steam-not-yanked");
         }
     }
 
-    function testYankDAILegacy() private { // make private to disable
+    function testYankDAILegacy() private {
```

Comment:
What is the reason of keeping some tests `private` instead using new modifier? I think this adds additional layer of complexity and we should either mark all of the private ones as skipped or just remove them (they would still be searchable in the archive like any other rarely used tests).

Reply:
I forgot to flag this one, but the idea is to eventually remove this one. I don't think there are any legacy DAI streams left, which means we would never use this again.

Reply:
Agree to remove this one. But there are 3 other private functions inside `DssSpell.t.sol`. Do you want to remove them as well?

Reply:
Done.

Reply:
So you don't plan to remove it in this PR? Fine with me. I just don't understand the convention between `private` vs `public skipped`. Why not keep `TODO`, but add `public skipped` instead of `private` until we remove them?

Reply:
Note: it was [my suggestion to the crafter](https://github.com/makerdao/spells-mainnet/pull/380#discussion_r1447411837) to add `testYankDAILegacy` as it didn't exist before 2024-01-12. Instead, there was always this comment: 
https://github.com/makerdao/spells-mainnet/blob/58203208eae6d939162d950197afa1014bf4d9c9/src/DssSpell.t.sol#L428

So it's definitely safe to remove it from my perspective

Reply:
`public skipped` means it's currently usable if required. All `private` are meant to be removed.

I wouldn't put `public skipped` in those one to not bloat the output of the test suite of tests that are no longer useful, but if you think that's a better approach, I see no harm in doing it that way.

Reply:
I removed `testYankDAILegacy` and replaced all `private` modifiers with `public skipped`.